lisa/sut_orchestrator/azure/features.py

```diff
@@ -1392,6 +1392,9 @@ def _select_vm_size(
         # Choose random size from the list to resize to
         index = randint(0, len(avail_eligible_intersect) - 1)
         resize_vm_size = avail_eligible_intersect[index]
+        node_runbook.vm_size = resize_vm_size.vm_size
```

Comment:
Can we set the property of the related test results, make sure both old/new vm sizes are logged? You can check if the environment can save this information, and bring to test results. It doesn't block this PR, just add a task to track it.

Reply:
How about resize back to the original vm size?

Reply:
I think it takes longer time than deploying a new one. It may cause other problems. So mark it dirty is clearer.

Reply:
updated